frontend/src/testing/data/transitionData.ts

```diff
@@ -5,17 +5,31 @@ export default createTransitions([
   ['43-4051', 'Customer service representatives', 0.0421, 17.9, 37300],
   ['11-9051', 'Food service managers', 0.0407, 28.76, 59820],
   ['35-3011', 'Bartenders', 0.039, 13.46, 28000],
-  ['43-6014', 'Secretaries and administrative assistants, except legal, edical, and executive', 0.0391, 18.84, 39180,],
+  [
+    '43-6014',
+    'Secretaries and administrative assistants, except legal, edical, and executive',
```

Comment:
should "edical" be "medical"?

Reply:
this is fixed in the latest version. Are we all set?